src/components/ArticleCard.jsx

```diff
@@ -1,11 +1,22 @@
 import React from "react";
 import { Grid, Image } from "semantic-ui-react";
 import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+
+const ArticleCard = ({ article, size }) => {
+  const dispatch = useDispatch();
```

Comment:
Dispatch is already declared in the App.jsx so not really needed here, maybe use props instead.